client/reader/stream/index.jsx

```diff
@@ -500,7 +511,19 @@ class ReaderStream extends Component {
 					renderLoadingPlaceholders={ this.renderLoadingPlaceholders }
 				/>
 			);
-			const sidebarContent = <ReaderListFollowedSites path={ path } />;
+
+			let sidebarContent;
+			if ( path.startsWith( '/tag/' ) ) {
```

Comment:
nit: I think in JSX it's more idiomatic to use the ternary `if` statements for generating JSX e.g. https://github.com/Automattic/wp-calypso/blob/trunk/client/reader/tag-stream/empty.jsx#L44